Python/Data loader for Health Monitorization

```diff
@@ -0,0 +1,44 @@
+# Data loader for Health monitarization
+
```

Comment:
Also, if possible, please add a document for this project in the documentation folder in the repo.

Reply:
sure